### DISCUSSION TITLE
New UI

### DISCUSSION THREAD

**Discussion by iBLiSSIN** [IceHacks/SurvivCheatInjector#808] [Ideas]
**Describe the feature you'd like**
This UI is better 

**Describe alternatives you've considered**
V3 should use it 

**Additional context**
I using this as a user script now 
![1DDB9238-0A18-46E9-9233-2A063AB0B0CC](https://user-images.githubusercontent.com/77667370/140027304-1b030d72-5322-499e-b8ce-62abce57f9e8.jpeg)

**Comment by TylerWhat** [comment]
where do you get this UI?

**Comment by IceHacks** [comment]
> where do you get this UI?

That looks like the old cheat

**Comment by TylerWhat** [comment]
ok

**Comment by iBLiSSIN** [comment]
> > where do you get this UI?
> 
> That looks like the old cheat

Yea, and I just remake it to work

**Comment by AsiaGuy** [comment]
bruh this UI is so small, you have to scroll more on the battlepass

**Comment by iBLiSSIN** [comment]
> bruh this UI is so small, you have to scroll more on the battlepass

But the new IceHacks UI not have it (the battle pass)

**Comment by IceHacks** [comment]
If you have the CSS file for this, I will happily look into adding it.

**Comment by iBLiSSIN** [comment]
> If you have the CSS file for this, I will happily look into adding it.

https://github.com/Samer-Kizi/Surviv.io_Darktheme The code is here
